src/common/fldout_nc.f90

```diff
@@ -1219,7 +1219,7 @@ subroutine get_varids(iunit, varid, ierror)
   if (ierror /= NF90_NOERR .and. .not. ierror == NF90_ENOTVAR) return
   ierror = nf90_inq_varid(iunit, "total_wet_deposition", varid%iwdt)
   if (ierror /= NF90_NOERR .and. .not. ierror == NF90_ENOTVAR) return
-  ierror = nf90_inq_varid(iunit, "total_wet_dry_deposition", varid%accwdt)
+  ierror = nf90_inq_varid(iunit, "total_wet_deposition", varid%accwdt)
```

Comment:
Doesn't make sense to have total_wet_deposition in 1220 and 1222. Shouldn't it be total_acc_wet_deposition?